### PR TITLE
Services-account-fixes

### PR DESCRIPTION
**Pull request by bmckercher123**
Per Alex, fix the Value for Services account How to, to reflect
gdoc final version.

- Can merge at any time
- Just wording change approved by Alex

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 476686af2f34b848e4b649ab27bb0f4788d231db

https://deploy-preview-2891--calico.netlify.com

File: master/security/service-accounts.md [hunk 1]
```diff
@@ -8,11 +8,14 @@ Use {{site.prodname}} network policy to allow/deny traffic for Kubernetes servic
 
 ### Value
 
-Kubernetes RBAC allows you to control which users are allowed to create and use service accounts. Combined with {{site.prodname}} network policy, you can control the security boundary between users who can provision service accounts, and those who use them.
+Using Calico network policy, you can leverage Kubernetes service accounts with RBAC for flexible control over how policies are applied in a cluster. For example, the Security team can have RBAC permissions to:
```
Comment (caseydavenport) [inline master/security/service-accounts.md]: Calico -> {{site.prodname}} throughout

Comment (caseydavenport) [reply master/security/service-accounts.md]: "Security team" -> "security team"

File: master/security/service-accounts.md [hunk 2]
```diff
@@ -8,11 +8,14 @@ Use {{site.prodname}} network policy to allow/deny traffic for Kubernetes servic
...
 
-For example, using Kubernetes RBAC, you can restrict permissions to provision service accounts to only the network security team, who can then write high-priority network policies that reference those service accounts.
+- Control which service accounts the Developer team can use within a namespace
```
Comment (caseydavenport) [inline master/security/service-accounts.md]: "Developer" -> "developer"

File: master/security/service-accounts.md [hunk 3]
```diff
@@ -8,11 +8,14 @@ Use {{site.prodname}} network policy to allow/deny traffic for Kubernetes servic
...
+- Write high-priority network policies for those service accounts (that the Developer team cannot override) 
 
-Additionally, when using Istio-enabled apps with {{site.prodname}} network policy, the cryptographic identity associated with the service account is checked (along with the network identity) to achieve two-factor authentication.
+The Network Security team can maintain full control of security, while selectively allowing Developer operations where it makes sense.
```
Comment (caseydavenport) [inline master/security/service-accounts.md]: "Network Security" -> "network security"

File: master/security/service-accounts.md (hunk 1 above)
Comment (bmckercher123) [reply master/security/service-accounts.md]: Fixed

File: master/security/service-accounts.md (hunk 2 above)
Comment (bmckercher123) [reply master/security/service-accounts.md]: Fixed

File: master/security/service-accounts.md (hunk 3 above)
Comment (bmckercher123) [reply master/security/service-accounts.md]: Fixed

**Pull request merged**